### PR TITLE
[TECH]create edit tests page

### PR DESCRIPTION
**Pull request by alesiadrazdova**

### CODE REVIEW

File: src/components/EditTestsSelector/EditTestsSelector.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import EditTestsInfo from './EeditTestsInfo/EditTestsInfo';
+import imageSrc from '../../assets/images/goose.svg';
+import './EditTestsSelector.scss';
+
+
```
Comment (mtalstykh-exadel) [inline src/components/EditTestsSelector/EditTestsSelector.jsx]: empty string

File: src/components/EditTestsSelector/EditTestsSelector.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const EditTestsSelector = () => {
+  return (
+    <div className='edit-tests-selector'>
+      <div>
+        <EditTestsInfo />
+      </div>
```
Comment (mtalstykh-exadel) [inline src/components/EditTestsSelector/EditTestsSelector.jsx]: may be we don't need extra div

File: src/components/EditTestsSelector/EditTestsSelector.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+const EditTestsSelector = () => {
+  return (
+    <div className='edit-tests-selector'>
+      <EditTestsInfo />
+      <div className='edit-tests-placeholder'>
+        <div className='edit-tests-selector-text'>Select the option at the top to see the questions.</div>
+        <img title='goose-img' alt='goose-img' src={imageSrc} className='picture' />
+      </div>
+    </div>
+
```
Comment (tensenrey) [inline src/components/EditTestsSelector/EditTestsSelector.jsx]: remove empty string

File: src/pages/editTests/editTests.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Layout from '../../components/layout/Layout';
+import EditTestsSelector from '../../components/EditTestsSelector/EditTestsSelector';
+
+const EditTests = () => {
+  return (
+    <Layout>
+      <EditTestsSelector />
+      {/* <div>
+        <QuestionData/>
```
Comment (tensenrey) [inline src/pages/editTests/editTests.jsx]: is it somewhere to be used?

File: src/components/EditTestsSelector/EeditTestsInfo/EditTestsInfo.jsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+import Select from '@material-ui/core/Select';
+import { useState } from 'react';
+import './EditTestsInfo.scss';
+import MenuItem from '@material-ui/core/MenuItem';
+
+const EditTestsInfo = () => {
+
+  const [level, setLevel] = useState('');
+  const [module, setModule] = useState('');
+  const [ID, setId] = useState('');
```
Comment (tensenrey) [inline src/components/EditTestsSelector/EeditTestsInfo/EditTestsInfo.jsx]: id of what?

Comment (tensenrey) [reply src/components/EditTestsSelector/EeditTestsInfo/EditTestsInfo.jsx]: we can have several id, please write logical names

File: src/components/EditTestsSelector/EeditTestsInfo/EditTestsInfo.jsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+  const testLevels = ['A1', 'A2', 'B1', 'B2', 'C1', 'C2'];
+  const testModules = ['Gramma', 'Listening', 'Essay', 'Speaking'];
+
+  const testLevelsList = testLevels.map((elem) => { return <MenuItem className='edit-tests-option' key={elem} value={elem}>{elem}</MenuItem>; });
```
Comment (tensenrey) [inline src/components/EditTestsSelector/EeditTestsInfo/EditTestsInfo.jsx]: use return (
   <code>
)

File: src/components/testLevelSelector/testLevelSelector.jsx [hunk 7]
```diff
@@ -38,7 +38,7 @@ const TestLevelsSelector = () => {
     }
   ];
 
-  const testLevelsDataArr = testLevelsData.map((level, id) => <TestLevelsSelectorItem title={level.title} description={level.description} key={id}/>);
+  const testLevelsDataArr = testLevelsData.map((level, id) => <TestLevelsSelectorItem title={level.title} description={level.description} key={id} />);
```
Comment (tensenrey) [inline src/components/testLevelSelector/testLevelSelector.jsx]: use return (
)

**Review by mtalstykh-exadel — DISMISSED** [review]

**Review by tensenrey — APPROVED** [review]

**Review by mtalstykh-exadel — APPROVED** [review]

**Pull request merged**